app/assets/javascripts/version.ts

```diff
@@ -0,0 +1,8 @@
+/** Declared in webpack config */
+declare const __VERSION__: string;
+declare const __DESKTOP__: boolean;
+declare const __WEB__: boolean;
+
+export const appVersion = __VERSION__;
```

Comment:
Hmm I would make these PascalCase to indicate a global constant rather than var or function.
```suggestion
export const AppVersion = __VERSION__;
```

Reply:
Correct, will update. Do you think we should make `isDesktopPlatform` and `isWebPlatform` Pascal case as well? I think based on their name it's better to keep them camel cased, but I'm open for other opinions :)

Reply:
Yup same thing for those as well since they are global constants.

Reply:
Updated)